doc/src/static/red.css

```diff
@@ -24,8 +24,25 @@
   color: white;
 }
 
-body, h1, h2 {
-  font-family: 'PragmataPro';
+.wy-menu-vertical li a:hover {
+  background-color: #ee7977 !important;
+  color: white !important;
+}
+
+
+body, h1, h2, h3, h4, h5, h6, pre, code {
+  font-family: 'PragmataPro' !important;
+}
+
+pre {
+  padding: 12px 12px 0px 12px !important;
+}
```

Comment:
I feel there's too much space above the pre box. @jonsterling What do you think?

Reply:
you may be right! feel free to experiment and improve it...

Reply:
It's actually the bottom margin of <p> that's causing the problem.

Reply:
I'll leave it alone for the moment... I don't have enough brainpower to adjust it without destroying other parts.

Reply:
@favonia I know how you feel :laughing: